From 1bae1b8fbb65e605699dea83f268bbf9dfbd77d0 Mon Sep 17 00:00:00 2001
From: Richard Zhu <r65037@freescale.com>
Date: Wed, 8 Feb 2012 14:58:41 +0800
Subject: ENGR00174033-2 MX6 PCIE: add pcie RC driver

Add PCIE RC driver on MX6 platforms.
Based on iwl4965agn pcie wifi device, verified the following
features.
* Link up is stable
* map the CFG, IO and MEM spaces, and CFG/MEM spaces can accessed

Signed-off-by: Richard Zhu <r65037@freescale.com>
---
 drivers/pci/Kconfig |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/pci/Kconfig b/drivers/pci/Kconfig
index 0fa466a..565ab52 100644
--- a/drivers/pci/Kconfig
+++ b/drivers/pci/Kconfig
@@ -91,3 +91,5 @@ config PCI_IOAPIC
 config PCI_LABEL
 	def_bool y if (DMI || ACPI)
 	select NLS
+
+source "drivers/pci/pcie/Kconfig"
-- 
1.7.9.5

